src/service.ts

```diff
@@ -10,37 +10,50 @@ import { Variables } from './variables';
 
 import { Callstack } from './callstack';
 
-export class DebugService {
-  constructor(session: DebugSession | null, debuggerModel: Debugger.Model) {
-    this.session = session;
+export class DebugService implements IDebugger.IService {
+  constructor(debuggerModel: Debugger.Model) {
+    this._session = null;
```

Comment:
Should we keep the previous comment about keeping the session to `null` here too?

Reply:
Ah good catch!